cmd/prober/prometheus.go

```diff
@@ -49,6 +49,22 @@ var (
 	},
 		[]string{endpointLabel, hostLabel, statusCodeLabel, methodLabel})
 
+	grpcMethodLatenciesSummary = prometheus.NewSummaryVec(
+		prometheus.SummaryOpts{
+			Name:       "api_endpoint_latency",
+			Help:       "API endpoint latency distributions (milliseconds).",
+			Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001, .999: 0.0001},
+		},
+		[]string{hostLabel, methodLabel},
+	)
+
+	grpcMethodLatenciesHistogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "api_endpoint_latency_histogram",
+		Help:    "API endpoint latency distribution across Rekor and Fulcio (milliseconds)",
+		Buckets: []float64{0.0, 200.0, 400.0, 600.0, 800.0, 1000.0},
+	},
+		[]string{hostLabel, methodLabel})
```

Comment:
Any other metrics I could add here?

Reply:
Can we reuse the existing metrics? The method name should be enough to distinguish, or you could also prepend `grpc://` to the host.